lang/java/maven-plugin/pom.xml

```diff
@@ -24,7 +24,7 @@
     <artifactId>avro-parent</artifactId>
     <groupId>org.apache.avro</groupId>
     <version>1.12.0-SNAPSHOT</version>
-    <relativePath>../</relativePath>
```

Comment:
Just for info, is this a best practice?

Reply:
At least Intellij IDEA suggests so. It shows `../` as a warning.
I'm OK to revert it.